common_benchmark_suite/openxla/benchmark/comparative_suite/jax/scripts/generate_model_artifacts.sh

```diff
@@ -32,6 +34,7 @@ TD="$(cd $(dirname $0) && pwd)"
 VENV_DIR="${VENV_DIR:-jax-models.venv}"
 PYTHON="${PYTHON:-"$(which python)"}"
 WITH_CUDA="${WITH_CUDA:-}"
+AUTO_UPLOAD="${AUTO_UPLOAD:-}"
```

Comment:
(The convention from iree): Maybe we can have
```suggestion
AUTO_UPLOAD="${AUTO_UPLOAD:-0}"
```

And in below

```sh
if (( AUTO_UPLOAD == 1 )); then
```

Source: https://github.com/openxla/iree/blob/7d4abf0e201e18277edf207a27132d461bcc87bf/build_tools/cmake/ctest_all.sh#L74

Reply:
Sounds good. I was just following the `WITH_CUDA` convention, but the IREE version looks more straightforward.